<?xml version="1.0"?>
<launch>

  <!-- april analytical pick object script using simple_pregrasp_planner -->
  <node pkg="april_pick_place_object" type="pick.py" name="pick_object_node" output="screen" >
    <param name="import_file" type="string" value="simple_pregrasp_planner" />
    <param name="import_class" type="string" value="SimpleGraspPlanner" />
    <rosparam file="$(find april_pick_place_object)/config/pick_params.yaml" command="load" />
    <rosparam file="$(find april_pick_place_object)/config/grasp_planning_core_params.yaml" command="load" />
    <!-- pregrasp planner parameters -->
    <rosparam file="$(find april_pick_place_object)/config/grasp_planner/simple_pregrasp_planner_params.yaml" command="load" />
  </node>

</launch>
